beacon-chain/types/block.go

```diff
@@ -196,6 +203,12 @@ func (b *Block) IsValid(
 		}
 	}
 
+	cStateProposerRandaoSeed := cState.Validators()[proposerIndex].RandaoCommitment
+	if !b.isRandaoValid(cStateProposerRandaoSeed) {
+		log.Debugf("Invalid proposer RANDAO commitment. Wanted: %s. Got: %s", b.RandaoReveal(), cStateProposerRandaoSeed)
```

Comment:
Are these bytes? If so let's try 0x%x for formatting

Reply:
we've switched to `%#x` :)